repair_shards.c

```diff
@@ -186,11 +179,13 @@ RecreateTableDDLCommandList(Oid relationId, int64 shardId)
 	/* build appropriate DROP command based on relation kind */
 	if (relationKind == RELKIND_RELATION)
 	{
-		appendStringInfo(extendedDropCommand, DROP_REGULAR_TABLE_COMMAND, shardName);
+		appendStringInfo(extendedDropCommand, DROP_REGULAR_TABLE_COMMAND,
+						 quote_identifier(shardName));
```

Comment:
We quote the ``shardName`` just before this ``if`` block. Do we need to call ``quote_identifier`` again?